styles/_mixins.scss

```diff
@@ -14,8 +14,12 @@
   }
 }
 
-@mixin chatOpeningTransition($property) {
-  transition: $property 0.5s ease;
+@mixin chatOpeningTransition($props...) {
+  $transitions: ();
+  @each $prop in $props {
+    $transitions: append($transitions, #{$prop 0.5s ease}, comma);
+  }
+  transition: $transitions;
```

Comment:
I used multiple properties with the transition mixin at one point, but took them out. I left the enhancement to the mixin in place, it will be useful if later we need to apply this transition to more than one property.